DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/wageDataPayTypeFormTemplate.html

```diff
@@ -395,4 +395,3 @@
     </div>
   </div>
 </div>
-<h3 id="textAnchor">This is a test anchor.</h3>
```

Comment:
Are we sure we don't want to leave this?  How will people know that the test anchor works as expected?  😆